### PR TITLE
Styling fixes

### PR DESCRIPTION
**Pull request by kryptt**
making a pr from jan as it was on the wrong destination

### CODE REVIEW

File: dictionary.txt [hunk 1]
```diff
@@ -235,3 +235,123 @@ webkit-sticky
 webkit-text-fill-color
 whatsApp
 xmlns
+isHeight100Vh
+mobilePadding
+.swiper-slide-thumb-active
+ReactComponent
+RightArrorCircle
+StyledRightArrorCircle
```
Comment (kryptt) [inline dictionary.txt]: StyledRightArrorCircle -> StyledRightArrowCircle

File: dictionary.txt [hunk 2]
```diff
@@ -235,3 +235,123 @@ webkit-sticky
...
+ButtonWrapper
+maxWidth840
+maxWidth1140
+ViewportWrapper
+LightLinesBg
+FilledSquaresBg
+FirstLineWrapper
+eslint-disable
+linesBg
+LinesBg
```
Comment (kryptt) [inline dictionary.txt]: is it supposed to be both lower case and upper case?

File: dictionary.txt [hunk 3]
```diff
@@ -235,3 +235,123 @@ webkit-sticky
...
+straightSquares
+LinesWrapper
+IconWrapper
+mb-3
+hoverTransition
+useRef
+useOnScreen
+isElementOneVisible
+setIsElementOneVisible
+isElementTwoVisible
```
Comment (kryptt) [inline dictionary.txt]: this sounds like you hould be indexing and not have ElementOne and ElementTwo

**Review by kryptt — COMMENTED** [review]
No strong changes, mostly questions... I hadn't seen this PR because it was on the other repo.

There are couple of things I didn't like, but then they are things I dislike about react hooks more generally....

File: dictionary.txt (hunk 2 above)
Comment (JanJaroszczak) [reply dictionary.txt]: Yes. 'linesBg' is a svg file and 'LinesBg' is a styled React component.

File: dictionary.txt (hunk 1 above)
Comment (JanJaroszczak) [reply dictionary.txt]: I'll correct it.

**Pull request merged**